docs/fluttertodostutorial.md

```diff
@@ -1,483 +1,1488 @@
+
 # Flutter Todos Tutorial
 
 ![advanced](https://img.shields.io/badge/level-advanced-red.svg)
 
 > In the following tutorial, we're going to build a Todos App in Flutter using the Bloc library.
 
 ![demo](./assets/gifs/flutter_todos.gif)
+> ZZTODO: Update gif with the movie in the issue tracker.
 
 ## Key Topics
 
 - Observe state changes with [BlocObserver](/coreconcepts?id=blocobserver).
 - [BlocProvider](/flutterbloccoreconcepts?id=blocprovider), Flutter widget which provides a bloc to its children.
 - [BlocBuilder](/flutterbloccoreconcepts?id=blocbuilder), Flutter widget that handles building the widget in response to new states.
-- Using Bloc instead of Cubit. [What's the difference?](/coreconcepts?id=cubit-vs-bloc)
+- [BlocListener](/flutterbloccoreconcepts?id=bloclistener), a Flutter widget which invokes the listener code in response to state changes in the bloc.
+- Using 3 Blocs and  1 Cubit. [What's the difference?](/coreconcepts?id=cubit-vs-bloc)
+- Uses a [Repository and API layer](architecture/repository) separately for the data.
+- [RespositoryProvider](/flutterbloccoreconcepts?id=respositoryprovider), a flutter widget to provide a repository to its children.
 - Prevent unnecessary rebuilds with [Equatable](/faqs?id=when-to-use-equatable).
-- [MultiBlocProvider](/flutterbloccoreconcepts?id=multiblocprovider), a Flutter widget that merges multiple BlocProvider widgets into one.
+- [MultiBlocListener](/flutterbloccoreconcepts?id=multibloclistener), a Flutter widget that merges multiple BlocListener widgets into one.
+
 
 ## Setup
 
 We'll start off by creating a brand new Flutter project
 
-[script](_snippets/flutter_todos_tutorial/flutter_create.sh.md ':include')
+```shell
+flutter create flutter_login
```

Comment:
Can we revert this change? The previous script should still be correct and it should be `flutter create flutter_todos` not `flutter_login`.

Reply:
I think we should mention that the project was built using `very_good_cli` to redirect them to that package documentation in case they have questions about specific features that the CLI sets for the project, wdyt?